src/entrypoints/core.js

```diff
@@ -11,11 +11,18 @@ import { subscribePanels } from '../data/ws-panels.js';
 import { subscribeThemes } from '../data/ws-themes.js';
 import { subscribeUser } from '../data/ws-user.js';
 
-window.hassAuth = getAuth({
-  hassUrl: `${location.protocol}//${location.host}`,
-  saveTokens,
-  loadTokens: () => Promise.resolve(loadTokens()),
-});
+const hassUrl = `${location.protocol}//${location.host}`;
+
+if (location.search.includes('external_auth=1')) {
```

Comment:
Should we check for existence of native interface here before deciding to use external or just trust it exists and allow console error?

Reply:
If the URL parameter is set, we should assume we're in external auth mode